proposals/0173-Read-Generic-Network-Signal-data.md

```diff
@@ -16,9 +16,13 @@ This proposal is to make vehicle data APIs (GetVehicleData/SubscribeVehicleData/
 
 ## Proposed solution
 
+#### Definitions for terms used in proposal:
+* OEM Network Mapping: This file referse to key value pairs that the OEM module will use to retrieve vehicle data off its network
+* Vehicle Data Schema: This is a JSON object that contains vehicle data defintions for both requests and responses that can be sent using
```

Comment:
>vehicle data defintions

Please update to "vehicle data definitions"

Reply:
Done